v3/src/components/case-table/case-table.scss

```diff
@@ -56,15 +56,16 @@ $table-body-font-size: 8pt;
           .collection-title {
             position: absolute;
             text-align: center;
-            font-family: "museo-sans", sans-serif;
-            font-size: 12px;
+            font-family: "Noto Sans", sans-serif;
+            font-size: 14px;
+            font-weight: 500;
             text-overflow: ellipsis;
             cursor: text;
             vertical-align: middle;
             white-space: nowrap;
             user-select: none;
             font-style: italic;
-            color: #0f2633; // Not used anywhere else at the moment, so we're leaving it unnamed
+            color: black;
```

Comment:
My reading of the spec has the color of the title bar text as `#222`, not black. Probably worth a named constant since it occurs elsewhere.